projects/plugins/crm/admin/activation/welcome-to-jpcrm.php

```diff
@@ -267,7 +267,7 @@
 
 					<div style="width:100%;">
 						<div style="width:25%;float:left;">
-							<div class='yesplsess'><p><?php esc_html_e( 'Usage Tracking', 'zero-bs-crm' ); ?> <input type="checkbox" id="zbs_ess" value="zbs_ess" checked='checked'/></p></div>
+							<div class='yesplsess'><p><?php esc_html_e( 'Usage Tracking', 'zero-bs-crm' ); ?> <input type="checkbox" id="zbs_ess" value="zbs_ess"/></p></div>
```

Comment:
I know this is the existing wording, however it seems a little redundant having both the title 'Usage tracking' and the text next to the checkbox being the same. Maybe something like the following could work better:

```suggestion
							<div class='yesplsess'><p><?php esc_html_e( 'Track CRM Usage', 'zero-bs-crm' ); ?> <input type="checkbox" id="zbs_ess" value="zbs_ess"/></p></div>
```